crates/bounded_planet_core/src/networking/systems.rs

```diff
@@ -110,28 +113,21 @@ pub fn send_net_events(mut session: ResMut<SessionEventListenerState>, send_even
     // Publish packets ready to send to appropriate MPSC channels
     for send in session.send_event_reader.iter(&send_events) {
         match send {
-            SendEvent::SendPacket { connection_id, stream_id, data } => {
-                let sender = session
-                    .stream_senders
-                    .entry(*connection_id)
-                    .or_default()
-                    .get(&stream_id);
-
-                if let Some(sender) = sender {
-                    if let Err(e) = sender.send(Arc::clone(data)) {
+            SendEvent::SendPacket { connection: connection_id, stream: stream_type, data } => {
```

Comment:
Might be better to just change the local bindings to the same name as the struct variant's field names. If those names don't convey the point well, might be good to change them too.

For example: 
```suggestion
            SendEvent::SendPacket { connection, stream, data } => {
```